### PR TITLE
Feat/create api delegation receipt page

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
This PR adds ReceiptPage for ApiDelegation

## Related Issue(s)
- #80 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/rtk/features/delegationRequest/delegationRequestSlice.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  ],
+  failedApiDelegations: [
+    {
+      orgName: 'Org blabla',
+      apiName: 'Api blabla',
+    },
+  ],
+};
+
+export const delegateApi = async (
```
Comment (allinox) [inline src/rtk/features/delegationRequest/delegationRequestSlice.ts:41]: I'm assuming this is made for later use? Cause I can't see that it's used anywhere as of now

File: src/components/reusables/ConfirmationPage/ConfirmationPage.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    },
+  );
+
+  const successfulDelegatedItems = successfulDelegations?.map(
+    (apiDelegation: ApiDelegation, index: Key | null | undefined) => {
+      return (
+        <CompactDeletableListItem
+          key={index}
+          removeCallback={
+            successfulDelegations.length > 1 ? () => dispatch(softRemoveOrg(apiDelegation)) : null
```
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply src/components/reusables/ConfirmationPage/ConfirmationPage.tsx:116]: Good catch!

File: src/components/reusables/ConfirmationPage/ConfirmationPage.tsx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+export interface ConfirmationPageProps {
+  delegableApis?: DelegableApi[];
+  delegableOrgs?: DelegableOrg[];
+  failedDelegations?: ApiDelegation[];
+  successfulDelegations?: ApiDelegation[];
+  restartProcessPath: string;
+  pageHeaderText: string;
+  topListText?: string;
+  bottomListText?: string;
+  bottomText?: string;
```
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply src/components/reusables/ConfirmationPage/ConfirmationPage.tsx:41]: Yup for lack of better words the component was named ConfirmationPage. Open for name suggestions :)

File: src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx [hunk 4]
```diff
@@ -34,17 +41,22 @@ export const CompactDeletableListItem = ({
       >
         <div className={classes.baseListItemContent}>
           {startIcon && (
-            <div className={classes.listItemIcon}>
+            <div className={cn(classes.listItemIcon, classes[`listItemIcon__${contentColor}`])}>
```
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx:44]: Nice catch

File: src/components/reusables/ConfirmationPage/ConfirmationPage.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply src/components/reusables/ConfirmationPage/ConfirmationPage.tsx:41]: We should maybe separate the pages in the future if we get a lot of headache with them being one page. But now I've already gone through the pain of making a whole page reusable and we should keep it as a proof of misconception(for when people say that every component should be reusable), like we talked about earlier

[the remark this answers is not in the document]
Comment (allinox) [reply src/components/reusables/ConfirmationPage/ConfirmationPage.tsx:41]: Agreed 🙂

File: src/rtk/features/delegationRequest/delegationRequestSlice.ts (hunk 1 above)
Comment (Albertlarsen) [reply src/rtk/features/delegationRequest/delegationRequestSlice.ts:41]: Is going ot be used for later. Needed to write that code to solve a potential hard problem.

File: src/rtk/features/delegationRequest/delegationRequestSlice.ts [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      }
+    }
+  }
+};
+
+const postApiDelegation = createAsyncThunk(
+  'delegationRequestSlice/delegateApi',
+  async ({ apiIdentifier, orgNr }: DelegationRequest) => {
+    return await axios
+      .get('/accessmanagement/api/v1/1337/resources/maskinportenschema')
```
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply src/rtk/features/delegationRequest/delegationRequestSlice.ts:71]: I'm making a task on doing this

**Pull request merged**